falguard.py

```diff
@@ -64,7 +65,11 @@ class Validator:
 
     def __init__(self, spec_path):
         with open(spec_path, 'r') as f:
-            spec_dict = json.loads(f.read())
+            ext = spec_path.split(".")[-1]
+            if ext.lower() == 'yml' or ext.lower() == 'yaml':
+                spec_dict = yaml.load(f.read())
+            else:
+                spec_dict = json.loads(f.read())
```

Comment:
Always try to avoid ifology. Looks like we can tidy things up with a map like this:
```python
deserializers = {
    'json': json.loads,
    'yaml': yaml.load,
    'yml': yaml.load,
}
```

Reply:
Good idea. Will implement